docs/zz_generated.seed.ce.yaml

```diff
@@ -574,6 +574,10 @@ spec:
         requests:
           cpu: 50m
           memory: 32Mi
+    # IPFamilies configures the IP families to use for the LoadBalancer service.
+    ipFamilies: null
```

Comment:
Not sure if we should leave this as is for demonstration purposes or redact this field from the example. WDYT @xrstf? I would like to avoid defaulting it to let's say `IPv4` or something since it's mostly determined by the configuration that you used to create the underlying kubernetes cluster.